charmcraft.yaml

```diff
@@ -59,8 +59,9 @@ config:
       type: string
       default: "f1"
       description: |
-        Name of the network interface handling communication over the F1 interface.
-        Host interface to use for the DU Network. If not specified, a bridge will be used.
+        Host interface to use for the F1 communication with the DU.
+        Used only in with the `macvlan` cni.
+        The corresponding macvlan interface needs to exist on the host.
```

Comment:
I have one concern regarding with the description. These interfaces names are also using in bridge mode not only in the macvlan cni.

Reply:
Maybe we need to change the this sentence `Used only in with the `macvlan` cni.`.

Reply:
I addressed this, let me know if this is more readable.